pkg/kubelet/kubelet_node_status.go

```diff
@@ -474,6 +474,17 @@ func (kl *Kubelet) recordNodeSchedulableEvent(node *v1.Node) error {
 // TODO(madhusudancs): Simplify the logic for setting node conditions and
 // refactor the node status condition code out to a different file.
 func (kl *Kubelet) setNodeStatus(node *v1.Node) {
+	// TODO(dixudx): remove this in v1.14
```

Comment:
rather than adding specific logic into this method, move this into a `nodestatus.RemoveOutOfDiskCondition` node status func and add to the list here:

https://github.com/kubernetes/kubernetes/blob/62e8a1bf40a22680fe19a84addad4972deb1d846/pkg/kubelet/kubelet_node_status.go#L521-L532

Reply:
also needs a unit test

Reply:
and could be commented to be removed in 1.16 (skip-level kubelet upgrades are permitted), or could be left indefinitely without harm

Reply:
@liggitt Done. PTAL.